.github/workflows/tests-smoke.yaml

```diff
@@ -150,7 +150,7 @@ jobs:
 
           kubectl wait -n kube-system --for=condition=Ready --all pod --timeout=5m
           # To make sure that cilium CRD is available (default timeout is 5m)
-          # https://github.com/cilium/cilium/blob/v1.13/operator/crd.go#L34
+          # https://github.com/cilium/cilium/blob/master/operator/crd.go#L34
```

Comment:
This doesn't seem correct for a v1.13 backport?

Reply:
(comment so it's not functional, maybe it's easier to just have this stay the same as master to minimize diff/conflicts in future)

Reply:
actually this was likely caused by me. I'll revert it